src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface OperationHandler {
+    void getOperation(String fruit, int quantity);
```

Comment:
The method `getOperation` is used to perform an operation, which might be misleading as 'get' typically implies retrieval rather than execution. Consider renaming this method to something more indicative of its action, such as `executeOperation` or `performOperation`, to improve code readability.